layouts/partials/notification.html

```diff
@@ -2,3 +2,4 @@
 <div class="notification is-warning mb-0">
   {{ $notification }}
 </div>
+  
```

Comment:
Obsolete change?

Reply:
I might not have `git add`ed it the last time... :facepalm: 

https://github.com/fluxcd/fluxcd.io/pull/25/files

Reply:
Somehow resolved magically in the previous PR then.